templates/demo/web3js-react-dapp-demo/src/App.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders React Docs link', () => {
```

Comment:
tests should be in a `/test` folder

Reply:
This structure was inherited from `create-react-app` and I tried to change as little as possible.